tests/endpoint/test-endpoint.sh

```diff
@@ -20,8 +19,14 @@ source $COMMON_FILE
 validate_host "$1"
 validate_port "$2"
 
-# Create endpoint app
-make EP_TA_HOST="$1" EP_TA_PORT="$2" legato
+set -euo pipefail
 
-# Run endpoint app test here
-endpoint/_build_endpoint/localhost/app/endpoint/staging/read-only/bin/endpoint
```

Comment:
Is running the endpoint app test no longer required?

Reply:
Is it better to enable the app test on the simulator target?

Reply:
I just want to make sure the testing is still required and where the testing script would be placed.